app/assets/javascripts/controllers/navigation/active_item_controller.js

```diff
@@ -0,0 +1,36 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class ActiveItemController extends Controller {
+  static targets = ["menu"];
```

Comment:
This target appears to be redundant. You could use `this.element` instead, and assume that we will add the controller to menus that require this functionality.